src/snapshot/tosvg.js

```diff
@@ -16,6 +16,9 @@ var Drawing = require('../components/drawing');
 var Color = require('../components/color');
 
 var xmlnsNamespaces = require('../constants/xmlns_namespaces');
+var DOUBLEQUOTE_REGEX = /"/g;
+var DUMMY_SUB = 'TOBESTRIPPED';
```

Comment:
hmm, looks like this is already a problem in master:
```
Plotly.relayout(gd,{'yaxis.title':'"TOBESTRIPPED"'})
```
on screen:
![screen shot 2017-05-17 at 5 42 03 pm](https://cloud.githubusercontent.com/assets/2678795/26177581/2d5f0eb4-3b28-11e7-88be-b7e4a3089633.png)

download:
![newplot 2](https://cloud.githubusercontent.com/assets/2678795/26177553/05722a3a-3b28-11e7-9629-3f1c73a2e6a1.png)

So you're not making a new bug, but we do need something better for this.